docs/topics/rn-new-features.adoc

```diff
@@ -18,14 +25,7 @@ To execute the rules, use the target `jakarta-ee`.
 
 link:https://docs.openrewrite.org/[OpenRewrite] automates large-scale, distributed source code refactoring. You can execute OpenRewrite recipes by using the link:{ProductDocUserGuideURL}[{ProductShortName} CLI].
```

Comment:
Nitpick: I think we agreed to change "execute" to "run". Must have missed this one when I reviewed it earlier.